packages/gas-fee-controller/package.json

```diff
@@ -55,7 +56,8 @@
     "typescript": "~4.8.4"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^13.0.1"
+    "@metamask/network-controller": "^13.0.1",
+    "@metamask/polling-controller": "^0.0.0"
```

Comment:
Welp. This shouldn't be a peerDependency in this case, but the contraints are enforcing it due to the name.

The intention was to enforce that _controllers_ are peerDependencies of each other. But the `polling-controller` package is not itself a controller, it's a mixin meant to be used by controllers.

Reply:
Not sure how to proceed here @Gudahtt ?

Reply:
We can add an exception for `polling-controller` to the constraints here: https://github.com/MetaMask/core/blob/a1e1883a376a2e8747fc3e389fb3866b3b663008/constraints.pro#L308

We've already added an exception for two other packages

Reply:
Ok I'll add that in this PR.